fmriprep/workflows/anatomical.py

```diff
@@ -75,10 +79,46 @@ def t1w_preprocessing(name='t1w_preprocessing', settings=None):
         ),
         name='T1_2_MNI_Registration'
     )
-    # should not be necesssary byt does not hurt - make sure the multiproc
+    # should not be necesssary but does not hurt - make sure the multiproc
     # scheduler knows the resource limits
     t1_2_mni.interface.num_threads = settings['ants_nthreads']
 
+    # 6. FreeSurfer reconstruction
+    autorecon1 = pe.Node(
+        freesurfer.ReconAll(
+            directive='autorecon1',
+            args='-noskullstrip -parallel -openmp {:d}'.format(cpu_count())),
```

Comment:
Also set the number of expected threads for the scheduler (See https://github.com/poldracklab/fmriprep/blob/master/fmriprep/workflows/anatomical.py#L253)